packages/data-provider/src/services/submissionService.ts

```diff
@@ -371,6 +371,27 @@ const service = (dependencies: BaseDependencies) => {
 				processedEntities: entitiesToProcess,
 			};
 		},
+
+		deleteActiveSubmissionById: async (submissionId: number): Promise<Submission | undefined> => {
+			const { getSubmissionById, update } = submissionRepository(dependencies);
+
+			const submission = await getSubmissionById(submissionId);
+			if (_.isEmpty(submission) || !submission.dictionaryId)
+				throw new BadRequest(`Submission '${submissionId}' not found`);
+
+			// list of considered Open statuses of a submission able to be closed
+			const openStatuses = [SUBMISSION_STATUS.OPEN, SUBMISSION_STATUS.VALID, SUBMISSION_STATUS.INVALID];
+
+			if (!_.includes(openStatuses, submission.status)) {
+				throw new StatusConflict('Submission does not have a valid status to be deleted');
```

Comment:
Only `Open/valid/invalid` statuses are considered as `Active Submission` and therefore can be deleted.
Other submission statuses like `committed` is not considered Active and can not be deleted.

Reply:
🤔 I may also add the current status into that error log, to help troubleshoot.
```suggestion
				// Only "open", "valid", and "invalid" statuses are considered Active Submission
				throw new StatusConflict('Only active submissions can be deleted');
```